### PR TITLE
test markdown.js

### PR DESCRIPTION
**Pull request by wandersonpsantos**
## What type of PR is this? (check all applicable)

- [ ] Refactor
- [ ] Feature
- [ ] Bug Fix
- [x] Enhancement
- [ ] Documentation Update

## Description
#299 

## QA Instructions, Screenshots, Recordings

_Please replace this line with instructions on how to test your changes, as well
as any relevant images for UI changes._

## Added to documentation?

- [ ] readme

### CODE REVIEW

File: src/components/__tests__/markdown.test.js [hunk 1]
```diff
@@ -0,0 +1,233 @@
+import React from "react"
+import renderer from "react-test-renderer"
```
Comment (Citrinin) [inline src/components/__tests__/markdown.test.js:2]: Please rely on `enzyme-to-json` library and do not use `react-test-renderer`  - there is no trace of it in `master` now. Can see [here](https://github.com/rahuldkjain/github-profile-readme-generator/blob/master/src/components/__tests__/work.test.js#L22)

**Review by Citrinin — CHANGES_REQUESTED** [review]
Thanks for your efforts, looks almost good! 
But we shall rely only on one testing library.
Please, rebase to master & get rid of `yarn-lock.json` changes (it does not exist in master)

**Review by Citrinin — APPROVED** [review]
LGTM! 👍 
Thank you for your effort.

**Review by rahuldkjain — APPROVED** [review]
Superb @wandersonpsantos

**Pull request merged**